<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>536</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>933</width>
    <height>536</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>933</width>
    <height>536</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TOTÓ MEU AMIGÃO - PET - CADASTRAR - EDITAR - EXCLUIR - LISTAR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnListarPets">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>460</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 216, 98);
border-style: outset;
border-width: 2px;
border-radius: 15px; 
border-color: rgb(126, 126, 126);</string>
    </property>
    <property name="text">
     <string>Listar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddPet">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 216, 98);
border-style: outset;
border-width: 2px;
border-radius: 15px; 
border-color: rgb(126, 126, 126);</string>
    </property>
    <property name="text">
     <string>Add Pet</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabelaPetGeralporCliente">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>721</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <column>
     <property name="text">
      <string>Nome Pet</string>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Idade</string>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Especie</string>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Porte</string>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Raça</string>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Peso</string>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Responsável</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnBuscar">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNomepet">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="placeholderText">
     <string>Digite o Nome do Cliente ou do Pet. </string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>891</width>
      <height>431</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>891</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Pet</string>
    </property>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>340</y>
       <width>891</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnExcluirpet">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>370</y>
       <width>140</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 216, 98);
border-style: outset;
border-width: 2px;
border-radius: 15px; 
border-color: rgb(126, 126, 126);</string>
     </property>
     <property name="text">
      <string>Excluir</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAlterarPet">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>370</y>
       <width>151</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 216, 98);
border-style: outset;
border-width: 2px;
border-radius: 15px; 
border-color: rgb(126, 126, 126);</string>
     </property>
     <property name="text">
      <string>Alterar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPdfPet">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>370</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 216, 98);
border-style: outset;
border-width: 2px;
border-radius: 15px; 
border-color: rgb(126, 126, 126);</string>
     </property>
     <property name="text">
      <string>Gerar PDF</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>btnListarPets</zorder>
   <zorder>btnAddPet</zorder>
   <zorder>tabelaPetGeralporCliente</zorder>
   <zorder>btnBuscar</zorder>
   <zorder>txtNomepet</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtNomepet</tabstop>
  <tabstop>btnBuscar</tabstop>
  <tabstop>tabelaPetGeralporCliente</tabstop>
  <tabstop>btnExcluirpet</tabstop>
  <tabstop>btnAlterarPet</tabstop>
  <tabstop>btnListarPets</tabstop>
  <tabstop>btnPdfPet</tabstop>
  <tabstop>btnAddPet</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
